group-coordinator/src/main/java/org/apache/kafka/coordinator/group/metrics/GroupCoordinatorMetricsShard.java

```diff
@@ -145,17 +133,12 @@ public void incrementNumOffsets() {
     }
 
     /**
-     * Increment the number of consumer groups.
+     * Set the number of consumer groups.
```

Comment:
Should we also mention that this should be the only way to update this map? also for setClassicGroupGauges